client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx

```diff
@@ -29,10 +29,10 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
     const [currentInteractedContact, setCurrentInteractedContact] = React.useState<InteractedContact>();
     const [familyRelationships, setFamilyRelationships] = React.useState<FamilyRelationship[]>([]);
 
-    const { saveContactQuestioning, saveContact, updateInteractedContact, changeIdentificationType, openAccordion, updateNoResponse, loadInteractedContacts } = useContactQuestioning({ interactedContactsState, setCurrentInteractedContact });
+    const { saveContactQuestioning, saveContact, updateInteractedContact, changeIdentificationType, openAccordion, updateNoResponse, getMinimalDateToInvestigate } = useContactQuestioning({ interactedContactsState, setCurrentInteractedContact });
 
     React.useEffect(() => {
-        loadInteractedContacts();
+        getMinimalDateToInvestigate();
```

Comment:
I think you should find a better name for this function. My first instinct is to think it returns minimal date to investigate but in reality it loads interacted c;ontacts by minimal date.

Reply:
fixed